src/routes/blog/post/announcing-csv-imports/+page.markdoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Announcing: Document imports from CSV files"
+description: Learn how to import documents into your Appwrite collections using a simple CSV file — a new feature built on top of our powerful migration APIs.
+date: 2025-04-20
+# update this cover later, once available!
+cover: /images/blog/planetscale-databases-alternative/cover.png
+timeToRead: 5
+author: darshan-pandya
+category: announcement
+featured: false
+---
+
+We're introducing a new way to populate your Appwrite databases: **document imports from CSV files**.
+
+Built on top of Appwrite's powerful migration APIs, this feature makes it easy to bring in large datasets, seed collections, or migrate structured data—using nothing more than a CSV file.
+
+## How it works
+
+To get started, create a collection and define its attributes in the Appwrite Console. Your CSV file should follow a standard format:
+
+- The first row must be a header containing attribute names that match your collection
```

Comment:
Let's add a screenshot where we share instructions for console actions. Please note that screenshots should follow our guidelines.